ether_ipApp/src/ether_ip_test.c

```diff
@@ -385,13 +405,14 @@ int main (int argc, const char *argv[])
         else
         {
            tag = EIP_parse_tag(argv[i]);
+           
         }
     }
     if (tag && EIP_verbosity >= 3)
     {
         char buffer[EIP_MAX_TAG_LENGTH];
         EIP_copy_ParsedTag(buffer, tag);
-        EIP_printf(3, "Tag '%s'\n", buffer);
+        EIP_printf(3, "Tag : %s\n", buffer);
```

Comment:
All good, I just wonder why you removed the `'%s'` quotes here. When a read/write fails, it can be helpful to see the tag with quotes to assert that you're looking at `'SomeTag'` and not by accident ended up with `' SomeTag'` that has a leading space. Since all your other changes are about making it more robust, I'd leave the quotes in here for the same reason.

Reply:
Reverted these quotes back. 

The reason why I did is that `'%s'` is the troublesome in shell environment, if I try to do test massively within shell environment. However, there are more problems I saw. Thus, it is not relevant anymore. since I cannot use shell environment with `ether_ip_test`. Anyway, I typically use `-%s-` instead of `'%s'`.